libraries/lib-wave-track/WaveTrack.h

```diff
@@ -941,6 +941,9 @@ class WAVE_TRACK_API WaveTrack final
       void StretchLeftTo(double t);
       void StretchRightTo(double t);
 
+      void ApplyStretchRatio(const std::function<void(double)>& reportProgress);
```

Comment:
Reusing the `ProgressReporter` alias would be better for consistency.

Reply:
I haven't used it ... for consistency sake :-)

I was just copying the WaveClip interface, where ApplyStretchRatio is defined as

```c++
   /*!
    * @brief Renders the stretching of the clip (preserving duration).
    * @post GetStretchRatio() == 1
    */
   void ApplyStretchRatio(
      const std::function<void(double)>& reportProgress);
```

Reply:
Oh really 😅 Well, do as you please, maybe I'll fix it up in some future PR.